xwiki-platform-core/xwiki-platform-oldcore/src/main/java/com/xpn/xwiki/web/EditForm.java

```diff
@@ -280,4 +282,34 @@ public void setHidden(String hidden)
         this.hidden = hidden;
     }
 
+	/**
+	 *
+	 * Return the object policy given in the HTTP request. See
+	 * {@link com.xpn.xwiki.web.ObjectPolicyType ObjectPolicyType} for more
+	 * information about what is an object policy.
+	 *
+	 * @return Return the Object Policy type
```

Comment:
actually it should be (without uppercase):

     @return the Object Policy type